apps/imb/reframe_imb.py

```diff
@@ -0,0 +1,119 @@
+""" Performance tests using Intel MPI Benchmarks.
+
+    See README.md for details.
+
+    For parallel-transfer tests (i.e. uniband and biband) which use multiple process pairs, the -npmin flag is passed so that only the specified number of processes is run on each test.
+    This is the easiest way of ensuring proper process placement on the two nodes.
+
+    Run using e.g.:
+
+        reframe/bin/reframe -C reframe_config.py -c apps/imb/ --run --performance-report
+"""
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import sys, os, math
+from collections import namedtuple
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+from modules.imb import read_imb_out
+from modules.reframe_extras import ScalingTest
+from modules.utils import identify_build_environment
+
+Metric = namedtuple('Metric', ['column', 'function', 'unit', 'label'])
+
+class IMB_base(rfm.RegressionTest):
+    METRICS = []
+
+    def __init__(self):
+        self.valid_systems = ['*:compute-node']
```

Comment:
Side note, I'm thinking that we may want to remove the login node partitions from the systems.  They're useful for _quick_ tests, but not much more than that.

Reply:
I'm inclined to agree, we shouldn't really be setting up things to run on login nodes